pack/templates/nix_env_gen.yml

```diff
@@ -12,7 +12,10 @@ steps:
   inputs:
     disableAutoCwd: true
     scriptPath: 'pack/scripts/nix_deps.sh'
+# The following task is disabled since Python 3.9 to fix azure/ namespace
+# https://github.com/Azure/azure-functions-python-worker/pull/854
```

Comment:
Will we re-enable this particular task later?

Reply:
So this is an actual fix for Python 3.9 and afterward.